.gitignore

```diff
@@ -1 +1,3 @@
 # See https://www.notion.so/serokell/gitignore-3d45b853c4494fa282dc04269f851b2e
+.vscode/
```

Comment:
As the link above says, do not:
> Include patterns for temporary or state files created by tools that you use, but others might not. A common example of such a tool is a text editor – there are many of them.

I think ` .vscode/` and ` *.code-workspace` fall into this category. It's better to put them into your global gitignore (the link above says how to do it).